docs/vgclog.md

```diff
@@ -0,0 +1,966 @@
+
+You can enable verbose GC logs by specifying the `-verbose:gc` standard option when you start your application. For more information, see [standard command-line options](cmdline_general.md). 
+
+The output of `-verbose:gc` is printed to STDERR by default. To print the log output to a file, append the `-Xverbosegclog` option. You can print the verbose GC log file to a succession of files, where each file logs a specified number of GC cycles, by using extra parameters of `-Xverbosegclog` option. For more information, see the command-line option [`-Xverbosegclog`](xverbosegclog.md). 
+
+## Verbose GC log contents  
+The Verbose garbage collection logs are printed in XML format and consist of the following sections:  
+
+- a summary of your GC configuration, which is captured in the `<initialized>` XML element. 
+
+- sections that contain information about the different GC cycles run, including the GC operations and GC increments that made up the GC cycle.  
+
+The logs record when a GC cycle or increment starts and ends, together with the GC operations that are run to manage or reclaim memory. You can also find out what triggers these activities and the amount of memory available to your application before and after processing.  
+
+Not all operations that take place during garbage collection are recorded. For example, the logs record when an increment of a concurrent GC cycle starts and ends, but most concurrent GC operations are not recorded.
+
+For definitions of GC cycles, GC increments, operations, phases, see [Garbage Collection Policies](gc.md#garbage-collection-policies). 
+
+## XML Structure 
+
+GC activities that make up a GC cycle are listed in the XML structure in a linear fashion. Some XML elements contain child XML elements while other XML elements are empty, serving as markers for starting and ending parts of the GC cycle. The XML elements contain attributes that record further information about the activity. 
+
+The start of a GC cycle is recorded by using the <cycle-start> XML element. The trigger reason for the GC cycle is captured in an adjacent XML element, such as <af-start> for an allocation failure or <allocation taxation> for a policy’s memory threshold trigger. 
+
+For example, the following XML structure can be found in the verbose GC logs that are generated from the `gencon` policy. In this example, the lines are indented to help illustrate the flow and some child XML elements are omitted for clarity: 
+
+```
+    <exclusive-start> </exclusive-start>
+
+      <af-start/> 
+
+        <cycle-start/> 
+
+          <gc-start> </gc-start> 
+
+            <allocation-stats> </allocation-stats> 
+
+              <gc-op> </gc-op> 
+
+          <gc-end> </gc-end> 
+
+        <cycle-end/> 
+
+        <allocation-satisfied/> 
+
+      <af-end/> 
+
+    <exclusive-end/> 
+```
+
+The `<af-start/>`, `<cycle-start/>`, `<cycle-end/>`, `<allocation-satisfied/>`, and `<af-end/>` XML elements are empty and contain only attributes. 
```

Comment:
not XML expert, but I think saying 'are empty' is a stronger statement than saying 'have no child elements'

Reply:
Thanks Aleks, will change.